code/game/objects/items/weapons/clown_items.dm

```diff
@@ -145,6 +145,14 @@
 	hitsound = 'sound/items/quack.ogg'
 	honk_delay = 10
 
+/obj/item/weapon/bikehorn/baton
```

Comment:
You can still honk at range with this.

There should probably be a way to remove the horn from the baton.